tests/tests/two-phase-migration.test.ts

```diff
@@ -0,0 +1,67 @@
+import { execa } from 'execa';
+import fse from 'fs-extra';
+import { beforeAll, describe, expect, test } from 'vitest';
+
+import { assertEmberTest, extractTests, migrate } from '../assertions.js';
+import {
+  type Project,
+  addonFrom,
+  build,
+  findFixtures,
+  install,
+  lintAddon,
+  lintTestApp,
+  makeMonorepo,
+} from '../helpers.js';
+
+let fixtures = await findFixtures();
+
+for (let fixtureName of fixtures) {
+  describe(`extract-tests + --exclude-tests on fixture: ${fixtureName}`, () => {
+    let project: Project;
+
+    beforeAll(async () => {
+      project = await addonFrom(fixtureName);
+      await extractTests(project.rootPath);
+      await makeMonorepo(project.rootPath);
+      await install(project.rootPath);
+      await migrate(project.packagePath, ['--exclude-tests']);
+      await build(project.packagePath);
```

Comment:
I took the freedom to refactor the test helpers a bit. They were coupled to the `Project` interface before, although they only ever needed a single path. Now you pass only that path, without having them to care about what a `Project` is. In this case this made them more flexible, as here we need to run `build` on `project.packagePath` (not `project.addonPath` as was hardcoded before)